include/votca/xtp/orbitals.h

```diff
@@ -1,4 +1,4 @@
-/* 
+/*
  *            Copyright 2009-2017 The VOTCA Development Team
```

Comment:
2009-2018

Reply:
Thanks again, I'm really bad at keeping the comments correct.